openassessment/xblock/peer_assessment_mixin.py

```diff
@@ -50,16 +51,6 @@ def peer_assess(self, data, suffix=''):
             and "msg" (unicode) containing additional information if an error occurs.
 
         """
-        # Validate the request
-        if 'options_selected' not in data:
-            return {'success': False, 'msg': self._('Must provide options selected in the assessment')}
-
-        if 'overall_feedback' not in data:
-            return {'success': False, 'msg': self._('Must provide overall feedback in the assessment')}
-
-        if 'criterion_feedback' not in data:
-            return {'success': False, 'msg': self._('Must provide feedback for criteria in the assessment')}
-
         if self.submission_uuid is None:
             return {'success': False, 'msg': self._('You must submit a response before you can peer-assess.')}
```

Comment:
I wonder how we're translating "peer-assess" as a verb? Suggest changing to "...before you can perform a peer assessment."